shell-scripts/setup-server.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+echo "Starting E2E test suit"
+
+if [[ -f "./shell-scripts/env.sh" ]]; then
+    echo "env.sh found - sourcing..."
+    # shellcheck disable=SC1091
+    source ./shell-scripts/env.sh
+    local_setup_repo=https://demo:$GITLAB_TOKEN@code.inmanta.com/inmanta/local-setup.git
+else
+    echo "No env.sh file found, checking for GITLAB_TOKEN variable instead"
+    if [ -n "$GITLAB_TOKEN" ]; then
+    echo "GITLAB_TOKEN successfuly found."
+    local_setup_repo=https://jenkins:$GITLAB_TOKEN@code.inmanta.com/inmanta/local-setup.git
+    else
+    echo "GITLAB_TOKEN variable not supplied."
+    exit 1
+    fi
+fi
+
+mkdir temp
+cd temp
+
+git clone $local_setup_repo || exit 1
+
+cd local-setup
+
+# This can be adjusted to pull in iso4 by replacing the command with 'yarn run pull:4dev'
+yarn run pull:5dev
```

Comment:
could actually maybe become a variable in the env.sh I guess?

Reply:
Right now there is no need for that, I guess, as there is only one script that use it, and in the future we can modify it.
But Also I don't mind if you would prefer to extract it :D

Reply:
Great ! I'll let it be for now, and keep it on a side note in my head for later :D thanks for your opinion :D